bin/sozo/src/commands/events.rs

```diff
@@ -43,21 +46,36 @@ pub struct EventsArgs {
 impl EventsArgs {
     pub fn run(self, config: &Config) -> Result<()> {
         let env_metadata = utils::load_metadata_from_config(config)?;
+        trace!(target: LOG_TARGET, "Fetched metadata from config");
+
         let ws = scarb::ops::read_workspace(config.manifest_path(), config)?;
+        trace!(target: LOG_TARGET, "Loaded workspace with {} members", ws.members().count());
+
         let manifest_dir = ws.manifest_path().parent().unwrap().to_path_buf();
+        trace!(target: LOG_TARGET, "manifest directory: {}", manifest_dir);
+
         let provider = self.starknet.provider(env_metadata.as_ref())?;
+        trace!(target: LOG_TARGET, "Provider: {:?}", provider);
 
         let event_filter = events::get_event_filter(
             self.from_block,
             self.to_block,
             self.events,
             self.world.world_address,
         );
-
+        trace!(
+            target: LOG_TARGET,
+            "Created event filter with from_block = {:?}, to_block = {:?}",
```

Comment:
We could add the chunk size here as it's often a source of weird behaviors depending the given value.